routers/user/home.go

```diff
@@ -194,7 +199,25 @@ func Issues(ctx *context.Context) {
 		page = 1
 	}
 
-	repoID := ctx.QueryInt64("repo")
+	reposQuery := ctx.Query("repos")
+	var repoIDs []int64
+	if issueReposQueryPattern.MatchString(reposQuery) {
+		// remove "[" and "]" from string
+		reposQuery = strings.Replace(strings.Replace(reposQuery, "[", "", 1), "]", "", 1)
+		//for each ID (delimiter ",") add to int to repoIDs
+		for _, rID := range strings.Split(reposQuery, ",") {
+			// Ensure nonempty string entries
+			if rID != "" && rID != "0" {
+				rIDint64, err := strconv.ParseInt(rID, 10, 64)
+				if err == nil {
+					repoIDs = append(repoIDs, rIDint64)
```

Comment:
I guess it's ok that there are duplicates in this list because our query will just handle these.

Reply:
I didn't test it jet but think so ... (testing ...)

Reply:
... later function take care of so dubble IDs is no problem

Reply:
Yes, this is "by design" (i.e. it could probably be done in another way, but using the later function gives desired result).